openmc/settings.py

```diff
@@ -1087,3 +1136,88 @@ def export_to_xml(self):
         tree = ET.ElementTree(self._settings_file)
         tree.write("settings.xml", xml_declaration=True,
                              encoding='utf-8', method="xml")
+
+
+class ResonanceScattering(object):
+    """Specification of the elastic scattering model for resonant isotopes.
+
+    Attributes
+    ----------
+    nuclide : openmc.nuclide.Nuclide
+        The nuclide affected by this resonance scattering treatment.
+    nuclide_0K : openmc.nuclide.Nuclide
+        This should be the same isotope as `nuclide`, but it should have an
+        `xs` attribute that identifies 0 Kelvin data.
+    method : str
+        The method used to sample outgoing scattering energies.  Valid options
+        are 'ARES', 'CXS' (constant cross section), 'DBRC' (Doppler broadening
+        rejection correction), and 'WCM' (weight correction method).
+    E_min : float
```

Comment:
To be consistent with the rest of the API, use `Real` in place of `float` here and for `E_max` below